src/modules/visits/dto/visits.dto.ts

```diff
@@ -63,14 +95,31 @@ export class VisitsQueryDto {
 			"Text to search for in the name or email af the requester. Use '%' for wildcard.",
 		default: '%',
 	})
-	query? = '%';
+	query?: string;
+
+	@IsUUID()
+	@IsOptional()
+	@ApiPropertyOptional({
+		type: String,
+		description: 'Get all visits for this user',
+	})
+	userProfileId?: string;
```

Comment:
add a TODO to auto fill this property with the user's own id for users that are not admins so they always only see their own visits.